extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -3,9 +3,6 @@
 class CommunityPageSpecialController extends WikiaSpecialPageController {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "CommunityPageSpecialController" has 22 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)